literature/jovic_1999_cyclic_247/jovic_1999_cyclic_247_p1_1_solid.yaml

```diff
@@ -0,0 +1,102 @@
+    version: 1
+    linked measurements: # FTIR, SXRD, Ring Disc - indicate if any other operando methods were shown in the same plot, which is linked to the data
+    comment: # Add whatever you think is important to understand the data
+
+electrochemical system:
+    version: 1
+    instrumentation:
+        - type:
+          supplier: 
+        - type:
+          model:
+          supplier:
+        - type:
+          model:
+          supplier:
+    electrolyte:
+        type: aq # aqueous = aq, non-aqueous=naq
+        components: # create as many components as necessary. A simple electrolyte consists of two components, i.e., H2SO4 and water
+            - name: NaOH # can be trivia name, sum formula, etc
+              concentration:
+                  value: 0.1
+                  unit: M #[M, mM, µM, g kg-1, ...]
+              type: #
+              source:
+                  supplier:
+                  LOT: 
+            - name: water
+              type: solvent
+              proportion:
+                  value:
+                  unit:
+              source:
+                  refinement:
+                  quality:
+                  total organic carbon: 
+                      value:
+                      unit:
+            - name: 
+              type:
+              proportion:
+                  value: 
+                  unit:
+            - name: 
+              type: 
+              source:
+                  supplier:
+                  LOT: 
+              concentration:
+                  value:
+                  unit: #[M, mM, µM, g kg-1, ...] 
+        ph: 
+            value:
+            uncertainty:
+        temperature: # provide the temperature as it is given in the article. Assume room temperature if nothing is provided
+            value: 298.15 
+            unit: K # [K, C, F]
+    electrodes:
+        configuration: 3 # number of electrodes
+        reference electrode:
+            type: SCE # the reference electrode given in the experimental section. Might differ from that in the plot, MSE-sat, MSE-3M, MSE-1M, RHE, AgCl2
+            source:
+                supplier:
+        counter electrode:
+            material: Pt
+            crystallographic orientation:
+            shape: sheet
+        working electrode:
+            material: Ag
+            crystallographic orientation: '111' # hkl, 100, 110, poly
+            source:
+                supplier:
+                LOT: 
+            shape: # optional
+                type: # bead/sphere
+                height:
+                    value:
+                    unit:
+                diameter:
+                    value:
+                    unit:
+            geometric electrolyte contact area: 
+                value:
+                unit:
+            preparation procedure: Ag crystal polished with fine emery paper, alumina, then etsched with cyanide and H2O2 # Short description of sample preparation. This can be very useful. :)

```

Comment:
```suggestion
            preparation procedure: Ag crystal polished with fine emery paper, alumina, then etched with cyanide and H2O2. # Short description of sample preparation. This can be very useful. :)

```